src/main/kotlin/org/bundleproject/Application.kt

```diff
@@ -2,10 +2,19 @@ package org.bundleproject
 
 import io.ktor.server.engine.*
 import io.ktor.server.netty.*
+import kotlinx.coroutines.runBlocking
 import org.bundleproject.plugins.*
+import org.bundleproject.utils.fetchAssets
 
 fun main() {
-    embeddedServer(Netty, port = 8080, host = "0.0.0.0") {
+    val portProp = System.getProperty("server.port")
```

Comment:
Instead could we use environment variables since this will be deployed via docker :D and to cleanup that function you can instead use System.getProperty(key)?.toIntOrNull() ?: 8080